lib/twilio/pstream.js

```diff
@@ -177,6 +177,22 @@ PStream.prototype.setToken = function(token) {
   this._publish('listen', payload);
 };
 
+PStream.prototype.sendMessage = function(
```

Comment:
Something to keep at the back of our heads. This pattern is old school and needs rework.